packages/core/src/lib/index.ts

```diff
@@ -19,6 +19,14 @@ export {
   default as TextInput,
   type TextInputTypes,
 } from './input/text-input.svelte';
+
 export { default as ContextMenu } from './context-menu/context-menu.svelte';
 export { default as ContextMenuItem } from './context-menu/context-menu-item.svelte';
 export { default as ContextMenuSeparator } from './context-menu/context-menu-separator.svelte';
+
+export { default as Table } from './table/table.svelte';
+export { default as THead } from './table/thead.svelte';
+export { default as TBody } from './table/tbody.svelte';
+export { default as TD } from './table/td.svelte';
+export { default as TH } from './table/th.svelte';
+export { default as TR } from './table/tr.svelte';
```

Comment:
looking for better ideas for these exported names 👀

Reply:
One of my favorite UI libraries is [Material UI](mui.com). I like how they name their table components - `Table`, `TableBody`, `TableRow`, etc: https://mui.com/material-ui/react-table/#api